neon_intrinsics/advsimd.rst

```diff
@@ -141,12 +141,16 @@ Document history
 +------------+-------------------------+-------------------------+
 |F           |30 May 2020              |Version ACLE Q2 2020     |
 +------------+-------------------------+-------------------------+
-|G           |30 October 2020          |Version ACLE Q2 2020     |
+|G           |30 October 2020          |Version ACLE Q3 2020     |
 +------------+-------------------------+-------------------------+
-|H           | |date-of-issue|         | |release|               |
+|H           |02 July 2021             |Version ACLE Q2 2021     |
```

Comment:
```suggestion
|H           |02 July 2021             |              2021Q2     |
```

I'd prefer consistency for the future, not with the past, and keep just one value everywhere: `2021Q2`.